.github/workflows/editorconfig.yml

```
@@ -38,4 +38,4 @@ jobs:
         cat "$HOME/changed_files" | nix-shell -p editorconfig-checker --run 'xargs -r editorconfig-checker -disable-indent-size'
     - if: ${{ failure() }}
       run: |
-        echo "::error :: Hey! It looks like your changes don't follow our editorconfig settings. Read https://editorconfig.org/#download to configure your editor so you never see this error again."
+        echo "::error :: Your changes do not follow editorconfig settings."
```

```suggestion
        echo "::error :: Your changes do not follow editorconfig settings. Read https://editorconfig.org/#download to configure your editor."
```

---

I don't need a tool to tell me where to put whitespace. **Fuck product placement.**

---

@ehmry please at least *try* to engage in a constructive and productive discussion.

-----

@lelgenio I'm not sure what's the benefit of directly linking to the downloads section. How about this instead:

```suggestion
        echo "::error :: Your changes do not follow editorconfig settings. See https://editorconfig.org/ for more information."
```